composer.json

```diff
@@ -22,12 +22,12 @@
         "cweagans/composer-patches": "^1.6.5",
         "drupal-composer/drupal-scaffold": "^2.5",
         "drupal/console": "^1.0.2",
-        "drupal/core": "^8.7",
+        "drupal/core": "^8.8",
```

Comment:
you don't need drupal/core and drupal/core-recommended - the new version brings core in as well.